### PR TITLE
Update DESCRIPTION.md with info on how to use SFTP

### PR DESCRIPTION
**Pull request by narFnarF**
## Problem

- The documentation didn't specify the SFTP username. See issue: https://github.com/YunoHost-Apps/my_webapp_ynh/issues/112

## Solution

- Add SFTP guidance

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
Let's use the new (v11.1) feature to display information in the webadmin.

**Review by tituspijean — APPROVED** [review]
To be merged after #107

File: doc/ADMIN.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Once logged in SFTP, under the Web directory, you will see a `www` folder which contains the public files served by this app. You can put all the files of your custom Web application inside.
+
+## Forgot your SFTP password?
+
+If you forgot your SFTP password, you can change it in Yunohost's webadmin interface in `Apps > My webapp > My Webapp configuration`.
+
+## PHP
+
+PHP-FPM version can be selected among 7.3, 7.4, 8.0, 8.1 and 8.2. To change it, use the configuration panel of the app.
```
Comment (alexAubin) [inline doc/ADMIN.md:21]: Nitpicking but i dunno if this bit really is useful, it just explains what should be visible one scroll further up in the webadmin interface

**Comment by ericgaspar** [comment]
closing with 3055c9a and eace720

**Pull request closed without merging**